WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -988,11 +1002,11 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
         }
     }
 
-    func keyForCurrentListStatusFilter() -> String {
+    class func keyForCurrentListStatusFilter() -> String {
         fatalError("You should implement this method in the subclass")
     }
 
-    func currentFilterIndex() -> Int {
+    class func currentFilterIndex() -> Int {
```

Comment:
PS: my take on these is that they should mostly be instance methods regardless of the underlying storage, which IMO could very well change to an instance variable.